apps/dashboard/config/locales/en.yml

```
@@ -173,6 +173,7 @@ en:
     not_grouped: "Other Apps"
 
     nav_limit_caption: 'showing %{subset_count} of %{total_count}'
+    featured_apps_title: "Apps"
```

```suggestion
    pinned_apps_category: "Apps"
```

This may make more sense as really we're talking about categories especially as it relates to the navbar.

---

I agree, changed.